{"uuid": "454a51a7-4ac6-44c4-9146-bbc4fcd0715c", "children": ["de5489a3-c6b0-4c62-b5d5-da8025d0fd66"], "befores": [{"name": "driver", "status": "passed", "start": 1744715427778, "stop": 1744715430317}], "afters": [{"name": "driver::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"C:\\Users\\Gebruiker\\Documents\\teachMeSkills\\flask\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Gebruiker\\Documents\\teachMeSkills\\flask\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n    ~~~~^^^^\n  File \"C:\\Users\\Gebruiker\\Documents\\teachMeSkills\\flask\\tests\\ui\\conftest.py\", line 22, in driver\n    config_driver.quit()\n    ~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Gebruiker\\Documents\\teachMeSkills\\flask\\venv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 193, in quit\n    self.service.stop()\n    ~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Gebruiker\\Documents\\teachMeSkills\\flask\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 157, in stop\n    self.send_remote_shutdown_command()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Gebruiker\\Documents\\teachMeSkills\\flask\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 142, in send_remote_shutdown_command\n    if not self.is_connectable():\n           ~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Gebruiker\\Documents\\teachMeSkills\\flask\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 131, in is_connectable\n    return utils.is_connectable(self.port)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\Gebruiker\\Documents\\teachMeSkills\\flask\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\utils.py\", line 101, in is_connectable\n    socket_ = socket.create_connection((host, port), 1)\n  File \"C:\\Users\\Gebruiker\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\socket.py\", line 856, in create_connection\n    exceptions.clear()  # raise only the last error\n    ~~~~~~~~~~~~~~~~^^\n"}, "start": 1744715440579, "stop": 1744715442606}], "start": 1744715427778, "stop": 1744715442606}